lib/async/als.js

```diff
@@ -6,6 +6,16 @@ import crypto from 'crypto';
 export const getId = () => crypto.randomUUID();
 
 export const MontiAsyncStorage = new AsyncLocalStorage();
+export const MontiAsyncIgnoreStorage = new AsyncLocalStorage();
+
+const oldBindEnv = Meteor.bindEnvironment;
+
+Meteor.bindEnvironment = function (...args) {
+  const func = oldBindEnv.apply(this, args);
+  return function () {
+    return MontiAsyncIgnoreStorage.run(true, () => func.apply(this, arguments));
```

Comment:
There should be a method on AwaitDetector to disable it for a function. This would remove the need for a separate store and be a little more efficient.